prow_config.yaml

```diff
@@ -180,6 +200,7 @@ workflows:
     job_types:
       - presubmit
     include_dirs:
+      - components/releasing/version/*
```

Comment:
Why is this line `components/releasing/version/*` while all the others are `components/releasing/version/*`?

Reply:
> Why is this line components/releasing/version/* while all the others are components/releasing/version/*?

I must be missing something, because both things look the same to me :P

Reply:
Ok I got what you mean!

Reply:
Sorry, copy/paste error. I thought the PR wasn't held so I jumped out of bed and looked through it very quickly.